buyer-api/.env.example

```diff
@@ -1,17 +1,21 @@
 NODE_ENV=development
 HOST=localhost
-PORT=9001
+PORT=9100 # bind port
 LOG_TYPE=combined
 CACHE=enabled
 CACHE_ADAPTER=memory
 ERROR_LOG='error.log'
 COMBINED_LOG='combined.log'
 
-BUYER_SIGNING_SERVICE_URL='http://localhost:9002'
-WEB3_PROVIDER='https://mainnet.infura.io/foobarbaz'
+# buyer-signing-service-api
+BUYER_SIGNING_SERVICE_URL='http://localhost:9101'
+# use local ganache server for development
+WEB3_PROVIDER='http://localhost:8545'
+# orders db, remember to flush when changing params
 REDIS_URL='redis://127.0.0.1:6379'
 REDIS_PREFIX='buyer-api'
 
+# addresses provided by log from # wibson-core/npm run truffle -- migrate --reset --compile-all
```

Comment:
I would split this in two lines so that it's clear how to get these address:
```
# addresses provided by npm script output from `wibson-core` repository:
# $ npm run truffle -- migrate --reset --compile-all
```

Reply:
that suits me even better ;) cheers.